### PR TITLE
COMMERCE-1581 fixed colums witdh on compare table with custom style

### PR DESCRIPTION
**Pull request by fabio-d-m**

### CODE REVIEW

**Comment by marco-leo** [comment]
ci:test

File: commerce-product-type-simple/src/main/resources/META-INF/resources/render/list_entry/compare_product.jsp [hunk 1]
```diff
@@ -92,16 +92,13 @@ CPSku cpSku = cpContentHelper.getDefaultCPSku(cpCatalogEntry);
...
-				</div>
 
-				<div class="autofit-col">
 					<liferay-commerce-cart:add-to-cart
 						CPDefinitionId="<%= cpCatalogEntry.getCPDefinitionId() %>"
 						CPInstanceId="<%= cpSku.getCPInstanceId() %>"
 						elementClasses="btn-block btn-primary text-truncate"
 						taglibQuantityInputId='<%= renderResponse.getNamespace() + quantityInputId %>'
 					/>
-				</div>
```
Comment (ethanbustad) [inline commerce-product-type-simple/src/main/resources/META-INF/resources/render/list_entry/compare_product.jsp]: @FabioDiegoMastrorilli don't we need this closing div tag? From what I see you're removing 1 opening tag and 2 closing tags. Or was the problem that it was lopsided before? But then we'd need to fix the indentation at least...

Comment (fabio-d-m) [reply commerce-product-type-simple/src/main/resources/META-INF/resources/render/list_entry/compare_product.jsp]: @ethanbustad you're right sir! Thank you for the check!

**Comment by marco-leo** [comment]
ci:test

**Comment by ethanbustad** [comment]
Just need to remove the trailing newline at the end of the file now.

**Comment by marco-leo** [comment]
@FabioDiegoMastrorilli please fix the SF issue as requested

**Comment by fabio-d-m** [comment]
@ethanbustad and @marco-leo  done it. Sorry about the inconvenience :(

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

**Pull request merged**